wget is broken

Log: https://gist.github.com/7ae7b09e9f7defb7c6e7
Debian Wheezy
Commit c9df66b1645f543f6644dfea6cda48c931dddcda + [changes in build-pkg](https://github.com/mxe/mxe/compare/master...LuaAndC:build-pkg-only-deps2)

I rebuilt `wget` for commit 4f2f9bf8064a0ad8733a2297d5a0cbcad7f35d3f without build-pkg and it fails with the same error. Maybe it fails on Debian Wheezy only.

The error message is "undefined reference to `libintl_dgettext'". It may be related to https://github.com/mxe/mxe/issues/1104

--- a/src/gnutls.mk
+++ b/src/gnutls.mk
@@ -8,7 +8,7 @@ $(PKG)_SUBDIR   := gnutls-$($(PKG)_VERSION)
 $(PKG)_FILE     := gnutls-$($(PKG)_VERSION).tar.xz
 $(PKG)_URL      := http://mirrors.dotsrc.org/gnupg/gnutls/v3.4/$($(PKG)_FILE)
 $(PKG)_URL_2    := ftp://ftp.gnutls.org/gcrypt/gnutls/v3.4//$($(PKG)_FILE)
-$(PKG)_DEPS     := gcc gettext gmp libgnurx nettle zlib
+$(PKG)_DEPS     := gcc gettext gmp libgnurx libidn nettle zlib
 
 define $(PKG)_UPDATE
     $(WGET) -q -O- ftp://ftp.gnutls.org/gcrypt/gnutls/v3.4/ | \

